src/components/grid-list/grid-tile.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  Component,
+  ViewEncapsulation,
+  Renderer,
+  ElementRef,
+  Input,
+} from '@angular/core';
+
+import {coerceToNumber} from './grid-list';
+
+@Component({
+  selector: 'md-grid-tile',
+  host: { 'role': 'listitem' },
+  templateUrl: './components/grid-list/grid-tile.html',
+  styleUrls: ['./components/grid-list/grid-list.css'],
+  encapsulation: ViewEncapsulation.None,
+})
+export class MdGridTile {
+  _rowspan: number = 1;
+  _colspan: number = 1;
+  _element: HTMLElement;
+  _isInitialized: boolean = false;
+
+  constructor(private _renderer: Renderer, element: ElementRef) {
+    this._element = element.nativeElement;
+  }
+
+  @Input('rowspan')
+  get rowspan() {
+  return this._rowspan;
```

Comment:
Indent, and you also don't need the `'rowspan'` string (same for colspan below).